raiden/log_config.py

```diff
@@ -102,6 +103,17 @@ def filter(self, record):
         return self._log_filter.should_log(record.name, record.levelname)
 
 
+def add_greenlet_name(logger, method_name, event_dict):
+    """
+    Add greenlet_name to the event dict for greenlets that have a non-default name.
+    """
+    current_greenlet = gevent.getcurrent()
+    greenlet_name = getattr(current_greenlet, 'name', None)
+    if greenlet_name is not None and not greenlet_name.startswith('Greenlet-'):
```

Comment:
why omit the `Greenlet-xxx` ones?

Reply:
Because those are not very informative since they are based on [`Greenlet.minimal_ident`](http://www.gevent.org/api/gevent.greenlet.html#gevent.Greenlet.minimal_ident) which recycles ids as soon as they become free again. That means that the same id/name can be used for completely different greenlets if they cycle fast-ish.

Reply:
I see. Thanks.